ml-agents/mlagents/trainers/brain.py

```diff
@@ -119,27 +118,6 @@ def __init__(
         self.agents = agents
         self.action_masks = action_mask
 
-    @staticmethod
-    @timed
-    def process_pixels(image_bytes: bytes, gray_scale: bool) -> np.ndarray:
```

Comment:
This was identical to the rpc_utils version. It now calls the new rpc_utils method.